inductiva/coastal/README.md

```diff
@@ -8,83 +8,61 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
+Note: You can follow this example in a notebook or run the script `test.py`
+available in this folder, that contains all the code snippets exhibited here.
+
 Start by creating or loading a bathymetry. This can be done e.g. by reading an
 [ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
 a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
-Here we shall use the bathymetry data from Algarve available [here](https://sextant.ifremer.fr/record/SDN_CPRD_590_HR_Lidar_Algarve/)
-as an example.
+Here we shall use a artificial bathymetry, randomly generated by our code.
 
+We start by generating a random bathymetry.
 ```python
 import inductiva
 
-bathymetry = inductiva.coastal.Bathymety.from_ascii_xyz_file(
-    ascii_xyz_file_path="590_HR_Lidar_Algarve.emo")
-```
-
-The bathymetry can be inspected by plotting it with a given resolution, in this
-case 200 m x 200 m:
-
-```python
-bathymetry.plot(x_resolution=200, y_resolution=200)
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(
+        x_range=(0, 100),
+        y_range=(0, 100),
+        x_num=100,
+        y_num=100,
+        max_depth=50)
 ```
 
-![Raw bathymetry.](resources/media/bathymetry.png)
-
-Raw bathymetry data will often span an area too large and sparsely sampled to
-be used in a simulation. In this case, the user can crop the bathymetry to a
-smaller area of interest:
+This bathymetry is defined in a 50x50 grid, that represent a domain 100 m
+wide and long. It can be inspected by plotting it.
 
 ```python
-bathymetry = bathymetry.crop(x_range=(51000, 52000), y_range=(12150, 13000))
 bathymetry.plot()
 ```
 
-![Cropped bathymetry.](resources/media/bathymetry_cropped.png)
+![Raw bathymetry.](/resources/bathymetry/bathymetry_random.png)
 
-To be used in a simulation, the bathymetry data must be interpolated to a
```

Comment:
Don't delete this example because it's a good one. It's just not the ideal first example. Pls just move it below to a section called "Simulating with real bathymetries", and make sure you can run it too.